bin/src/commands/launch/mod.rs

```diff
@@ -278,6 +278,32 @@ pub fn execute(matches: &ArgMatches) -> Result<Report, Error> {
             .join(" "),
     );
 
+    if let Some(mission) = launch.mission() {
+        let mut path = PathBuf::from(mission);
+
+        if !path.is_absolute() {
+            path = std::env::current_dir()?.join(mission);
+        }
```

Comment:
I would probably rather intentionally not allow absolute paths, and only missions in the .hemtt/missions folder or elsewhere in the project